index.md

```diff
@@ -58,44 +58,23 @@ Distinctions and Overlaps {#terms}
 
 ### Accessibility and Usability {#accessible-usable}
 
-While accessibility focuses on people with disabilities, many
-accessibility requirements also improve usability for everyone. For
-example, accessibility features such as providing sufficient contrast,
-scalable font size, and captioning also benefit people in limiting
-situations, such as using the web on a mobile phone when visual
-attention is elsewhere, in bright sunlight, in a dark room, and in quiet
-or noisy environments. Accessibility also addresses functional
-limitations aquired by age, even though some people may not identify
-these as "disability".
+Accessibility primarily focuses on people with disabilities. Many accessibility requirements improve usability for everyone, especially in limiting situations. For example, providing sufficient contrast benefits people using the web on a mobile device in bright sunlight or in a dark room. Captions benefit people in noisy and in quiet environments. Some people have age-related functional limitations, and may not identify these as "disability". Accessibility addresses these situations too.
```

Comment:
- Removed "But" from the start of the sentence
- Removed "color" from "contrast" (because it is actually luminosity contrast ratio, not color)
- Removed "elderly people", especially because "agre-related functional limitations already says that
- Removed second occurrence of "functional limitations"